MobiusExtras/Source/SimpleLogger.swift

```diff
@@ -20,36 +20,41 @@
 import Foundation
 import MobiusCore
 
-public class ConsoleLogger<Model, Event, Effect>: MobiusLogger {
+public class SimpleLogger<Model, Event, Effect>: MobiusLogger {
     private let prefix: String
+    private let consumer: Consumer<String>
 
-    public init(tag: String = "Mobius") {
+    public init(tag: String = "Mobius", consumer: @escaping Consumer<String> = { print($0) }) {
```

Comment:
On further consideration, it would be better to take a `TextOutputStream` and use `print(_:to:)`

Reply:
Turns out not to be a great idea